### ISSUE TITLE
Death Actions Does Not Work With TP Actions

### ISSUE THREAD

**Issue by size14pants** [PikaMug/Quests#1664]
**What is your Quests version / build number (do _not_ say "latest")?**

Quests 4.0.0-b218

**Describe the problem and how to replicate it. Any console errors to post _using a pastebin_?**

I made an action that teleports the player to a certain location. I made a quest that uses this as a Death Action so that upon death, the player is teleported to a specific location, however when a player dies, they are simply respawned in our default spawn and not where the action specified. This action works in everything action choice such as 'StartAction', so the action was not made incorrectly, I'm assuming that the 'Respawn' button overrides the action TP or maybe the player is teleported right before the death screen shows up. I don't know.

**If applicable, can you provide an example quest from quests.yml _using a pastebin_?**

**Here's an example quest:**

  custom17:
    name: Path of a Sorcerer - Part 1
    ask-message: Alright! To start, there's a curse on the second floor of this building.
      Take it out!
    finish-message: Well done!
    npc-giver-id: 88
    requirements:
      quests:
      - Enter Jujutsu Kaisen - Sorcerer Path!
    stages:
      ordered:
        '1':
          npc-ids-to-kill:
          - 90
          npc-kill-amounts:
          - 1
          death-event: 1stSorcererDeathTP
          start-message: Good luck!
          complete-message: You defeated the curse!
    rewards:
      money: 300
      quest-points: 1
      exp: 51
      commands:
      - questsadmin give <player> Get To Part 2
    planner:
      override: false
    options:
      allow-commands: true
      allow-quitting: true
      ignore-silk-touch: true
      use-dungeonsxl-plugin: false
      use-parties-plugin: true
      share-progress-level: 1
      same-quest-only: true

**As well as the action to go along with it:**

  1stSorcererDeathTP:
    teleport-location: Tokyo 801.0 7.0 -532.0

**Comment by PikaMug** [comment]
You are correct in that this is a limitation of Bukkit/Minecraft which denies players teleport at the death screen. I think this was touched on in #338 and, as you'll read, a solution is complicated. I may just end up disabling the ability to link teleport with Death Actions, but will leave this open pending further thoughts.
